test/cli/no-execution-summary-options.test.js

```diff
@@ -0,0 +1,58 @@
+var fs = require('fs'),
+
+    _ = require('lodash'),
+    sh = require('shelljs');
+
+describe('JSON reporter', function () {
```

Comment:
Similar to the comment to `test/library/` test, we should rename the `describe` and `it` blocks here.

Reply:
Fixed